flood_adapt/object_model/direct_impacts.py

```diff
@@ -4,86 +4,49 @@
 from flood_adapt.object_model.direct_impact.socio_economic_change import (
     SocioEconomicChange,
 )
-from flood_adapt.object_model.hazard.hazard import Hazard
-from flood_adapt.object_model.io.config_io import read_config, write_config
+from flood_adapt.object_model.hazard.hazard import Hazard, ScenarioModel
```

Comment:
The ScenarioModel schould be in the scenarios.py script? See respective comments in scenarios and hazard scripts

Reply:
That's exactly what I wanted to do, but I had an issue with cyclic imports when I tried that (basicaly the directimpacts uses the ScenarioModel for typing and it cannot be imported from a module with the class that contains it). I am actually thinking now that we should move all the pydantic class in the interface.